.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python:
   - "2.7"
-  - "3.4"
+  - "3.5"
```

Comment:
is there a reason to make this 3.5 instead of 3.6 or 3.7?

Reply:
I was keeping it up to date with zipline, but doesn't really have to be.

Reply:
Bumped it to 3.6 (travis didn't have 3.7)